corehq/apps/hqadmin/templates/hqadmin/email_status.html

```diff
@@ -0,0 +1,39 @@
+{% extends "hqwebapp/bootstrap3/base_section.html" %}
+{% load i18n %}
+
+{% block page_content %}
+  <p>{% trans "Check the status of an email" %}</p>
+  <form class="form-inline" method="GET" action="">
+    <input
+      name="q"
+      type="text"
+      class="form-control"
+      placeholder="you@example.com"
+    />
+    <input class="btn btn-primary" type="submit" />
```

Comment:
i prefer `button` elements, also better for translating text (which seems to be using the default)
```suggestion
    <button class="btn btn-primary" type="submit" >{% trans "Check Status" %}</button>
```

Reply:
ba2eefbbd82758fd2be344f5dc5c2f0e95a7141a